enderio-base/src/main/java/com/enderio/base/common/tag/EIOTags.java

```diff
@@ -60,6 +60,8 @@ private static void init() {}
         public static final IOptionalNamedTag<Block> FUSED_QUARTZ = BlockTags.createOptional(EnderIO.loc("fused_quartz"));
         public static final IOptionalNamedTag<Block> CLEAR_GLASS = BlockTags.createOptional(EnderIO.loc("clear_glass"));
         public static final IOptionalNamedTag<Block> DARK_STEEL_TIER = BlockTags.createOptional(EnderIO.loc("needs_dark_steel"));
+        public static final IOptionalNamedTag<Block> DARK_STEEL_EXPLODABLE_BLACKLIST = BlockTags.createOptional(EnderIO.loc("dark_steel_explodable_blacklist"));
+        public static final IOptionalNamedTag<Block> DARK_STEEL_EXPLODABLE_WHITELIST = BlockTags.createOptional(EnderIO.loc("dark_steel_explodable_whitelist"));

```

Comment:
a rename to denylist/blocklist and allowlist could be more clearer to people who never encountered these terms and be more descriptive in general about what they do